app/Filament/Chantier/Resources/Chantiers/Pages/ViewChantier.php

```
@@ -14,182 +45,232 @@ class ViewChantier extends ViewRecord
+                    DatePicker::make('started_at')
                         ->label('Date de début')
                         ->default(now())
                         ->required(),
-                    \Filament\Forms\Components\DatePicker::make('ended_at')
+                    DatePicker::make('ended_at')
                         ->label('Date de fin (Prévue)'),
                 ])
-                ->action(function (\App\Models\Chantiers\Chantier $record, array $data) {
-                    \App\Models\Flottes\VehicleAssignment::create([
+                ->action(function (Chantier $record, array $data) {
+                    VehicleAssignment::create([
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Flottes\VehicleAssignment::create().


(staticMethod.notFound)